content/applications/inventory_and_mrp/manufacturing/management/quality_control.rst

```diff
@@ -0,0 +1,80 @@
+================================
+Configure quality control checks
+================================
+
+Quality control is an essential part of the manufacturing process. Odoo's Quality module can be
+used to control the quality of products before they are registered into stock, during picking
+operations, and when leaving the warehouse in a delivery order. This is accomplished by creating
+*Quality Control Points* that will automatically trigger quality checks at specific points during
+production.
+
+Create and configure quality control points
+===========================================
+
+To create a quality control point, go to :menuselection:`Quality Control --> Control Points -->
+Create`. From here, configure the new control point based on the desired quality check workflow.
+
+.. image:: quality_control/quality-control-form.png
+   :align: center
+   :alt: Configure a quality control point in Odoo.
+
+Products
+--------
+
+Select the specific product or products that the check is performed on.
```

Comment:
Similar issue on this section and sections that follow: this is too lean.

This is a good example of _showing_, versus _telling_.

Headings, ideally communicate a specific outcome users want or expect, not just a UI label that they can already read for themselves. What about `Products` here? Can we cover specific use cases in this section? Or, after the bullet list overview at the top, is this section even necessary?